libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -23,16 +23,33 @@
 
 //! Mimic Qt's qApp for easy access to the application instance
 #define ccApp (static_cast<ccApplicationBase *>( QCoreApplication::instance() ))
-
+//! ccApplicationBase
+/*!
+ * ccApplicationBase provides a QApplication class with additional Constructor param isCommandLine and version
+ * The class also initialises openGL before instantiating the application class
+ * \see QApplication
+ */
 class CCAPPCOMMON_LIB_API ccApplicationBase : public QApplication
 {
 public:
 	//! This must be called before instantiating the application class so it
 	//! can setup OpenGL first.
 	static void	initOpenGL();
-	
+
+    //! ccApplicationBase Constructor
+	/*!
+	 * @param argc argc must be greater than zero, this is set by QT
```

Comment:
The description is simply: 'number of command line arguments' (and it could be 0, we don't really care?). This will be transferred **to** Qt by the caller of this constructor (it comes from the system).

Reply:
ok, no problem was re-phrased from the QT application ctor.